package.json

```
@@ -30,8 +29,12 @@
       "prettier --write",
       "git add"
     ],
+    "*.{json,yml,yaml}": [
+      "prettier --write",
+      "git add"
+    ],
     "*.md": [
-      "npm run lint:md-partial",
+      "npm run lint:md",
```

```suggestion
      "remark-preset-davidtheclark --format",
```

I think this should do it.

---

Updated the lint-staged and npm script to include the `--format` option